backend/prompt_studio/prompt_studio_registry/prompt_studio_registry_helper.py

```diff
@@ -283,6 +283,8 @@ def frame_export_json(
         tool_settings[JsonSchemaKey.ENABLE_HIGHLIGHT] = tool.enable_highlight
 
         for prompt in prompts:
+            if prompt.prompt_type == JsonSchemaKey.NOTES or not prompt.active:
```

Comment:
NIT: Abstract this check as a function if we're performing it at multiple places